### DISCUSSION TITLE
How do I display fields in a list view that aren't already showing?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#371] [FAQs] [answered]
How do I display fields in a list view that aren't already showing?

**Comment by fmjjones** [comment, marked as answer]
Any user can display any fields that are local to a module and summary strings from attached modules in list view. To do this you can select a pre-existing view by selecting View -> Choose list from the menu.

Search for some records.
From the top menu bar click View->List Setting->Choose List
Right click on the list view that you want to add the field too.
Select properties.
Scroll down the Fields box and double click on the field that you want to add. It will appear in the Columns box below.
Use the blue up and down arrows to move the field to the place you want it to be.
Click OK.
With the correct list view highlighted click OK
[This word document will show you in more detail how to create your own list views.](http://intranet.fieldmuseum.org/sites/default/files/Tips_ListViews.doc)
